tests/modules/aioseop/test-All_in_One_SEO_Pack-aiospMrtGetUrl.php

```diff
@@ -24,9 +24,12 @@
  * NOTE: The intent is to keep tests isolated.
```

Comment:
@EkoJr maybe this test case should be in tests/modules/**general** since it deals with the general settings? `aioseop` defines the entire plugin :)